collectd/files/python.conf

```diff
@@ -22,7 +22,13 @@
     {%- if array['variables'] is defined %}
     <Module "{{ module }}">
     {%- for key, value in array['variables'].items() | sort %}
-        {{ key }} {{ value }}
+        {%- if value is string or value is number %}
+          {{ key }} {{ value }}
+        {%- elif value is sequence %}
+          {{ key }} {{ value | json() }}
+        {%- else %}
+          {{ key }} {{ value }}
```

Comment:
This is the same as on line 26, you could therefore simply test for `is sequence` and have a single else branch.